### PR TITLE
components proposal

### PR DESCRIPTION
**Pull request by josmardias**
It's not ready for merging, just initiating a discussion on the topic.
This is a first draft of the Dashboard and it's components.

### CODE REVIEW

File: src/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+const Dashboard = (/* props */) => (
+  <div>
+    <Header />
+    <Body />
+  </div>
+)
+
+const Header = (/* props */) => (
+  <div>JOSMAR DIAS</div>
+)
```
Comment (josmardias) [inline src/dashboard.js]: This is the layout Header. I'm not sure if we will have some kind of "layout" or more than one page.
I think the header will be always the same.

File: src/dashboard.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+const Body = (/* props */) => (
+  <div>
+    <AccountsMenu />
+    <StatementControls />
+    <Statement />
+    <Summary />
+  </div>
+)
```
Comment (josmardias) [inline src/dashboard.js]: It's the body component of the dashboard.
Maybe we could call it "Page", or maybe just define everthing inside Dashboard.

File: src/dashboard.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+const StatementControls = (/* props */) => (
+  <div>
+
+  </div>
```
Comment (josmardias) [inline src/dashboard.js]: StatementControls is the component with date, search, pagination, etc...
screenshot from 2016-06-15 01-14-27

File: src/dashboard.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+)
+const AccountsMenu = (/* props */) => (
+  <div className="accounts">
+    <span className="tag">Itaucard</span><br />
+    <span className="tag nav">Bradesco</span><br />
+    <span className="tag nav">NuBank</span><br />
+    <span className="tag nav">+</span><br />
+  </div>
+)
```
Comment (josmardias) [inline src/dashboard.js]: left side vertical menu to chose account or add new one
screenshot from 2016-06-15 01-15-56

File: src/dashboard.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+const Dashboard = (/* props */) => (
+  <div>
+    <Header />
+    <Body />
+  </div>
+)
```
Comment (josmardias) [inline src/dashboard.js]: This is the Dashboard component. I'm breaking it in other components, and they are described below, but it's only a draft.
We need to agree on which components we need, and a file strutcture for them.

File: src/dashboard.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    <span className="tag nav">Bradesco</span><br />
+    <span className="tag nav">NuBank</span><br />
+    <span className="tag nav">+</span><br />
+  </div>
+)
+const Statement = (/* props */) => (
+  <div>
+
+  </div>
+)
```
Comment (josmardias) [inline src/dashboard.js]: Incomes and expenses list.
We can break it in some components like `<HumanReadableDate>`, `<TagsList>...`
screenshot from 2016-06-15 01-18-13

File: src/dashboard.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+const Summary = (/* props */) => (
+  <div>
+
+  </div>
+)
```
Comment (josmardias) [inline src/dashboard.js]: Overall balance on bottom of the statement

**Comment by vitormil** [comment]
👍

**Pull request merged**